drivers/include/net/netdev.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #include <sys/uio.h>
 
 #include "net/netstats.h"
+#include "net/l2filter.h"
```

Comment:
+1. Also: if there should be no name clashes in any way -> that's why we use the module name as prefix for everything a module defines...